src/content/components/ReleaseReport/ReleaseDatesTable.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
```

Comment:
Is this file being used anywhere?

Reply:
It's not, I forgot to delete it when I moved things into tabs!